### PR TITLE
add retry to azure wireserver APIs

### PR DESCRIPTION
**Pull request by xuhz**

### CODE REVIEW

**Comment by xuhz** [comment]
retest please

**Comment by xuhz** [comment]
retest this please

File: src/runtime_src/core/pcie/tools/cloud-daemon/azure/azure.h [hunk 1]
```diff
@@ -91,6 +91,7 @@ class AzureDev
     // 4 MB buffer to truncate and send
     static const int TRANSFER_SEGMENT_SIZE { 1024 * 4096 };
     static const int REIMAGE_TIMEOUT { 20 }; //in second
+    static const int UPLOAD_RETRY { 15 };
```
Comment (stsoe) [inline src/runtime_src/core/pcie/tools/cloud-daemon/azure/azure.h]: In general it is a bad idea to use upper cased variable names.  Though unlikely in this case, they are subject to perprocessor replacement

Comment (xuhz) [reply src/runtime_src/core/pcie/tools/cloud-daemon/azure/azure.h]: These variables are declared as static, and act as Macro defines.

**Pull request merged**